src/components/common/dropdown/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from "react";
+import { UseFormRegister, FieldValues, ValidationRule } from "react-hook-form";
+import styles from "./Dropdown.module.css";
+
+type DropdownProps = {
+  label: string;
+  options: Array<string>;
+  stylesName: string;
```

Comment:
```suggestion
  stylesName?: string;
```
I think this one should be optional, right? You're using a default value for `stylesName`

Reply:
True, updated!